shared/common-adapters/list.stories.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+import * as React from 'react'
+import Box, {Box2} from './box'
+import Text from './text'
+import List2 from './list2'
+import * as Sb from '../stories/storybook'
+import * as Styles from '../styles'
+
+const load = () =>
+  Sb.storiesOf('Common/List2', module)
+    .addDecorator(Sb.scrollViewDecorator)
+    .add('fixed - small', () => (
```

Comment:
Should not use the scroll view decorator.  This is a scroll view and we can't nest